tools/find-diff-or-untracked.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: AGPL-3.0-only
+
+set -eu -o pipefail
+
+function join_to_regex {
+    for i in "$@" ; do
+        echo -n "$i|"
+    done
+}
+
+function find_diff_or_untracked {
+    git diff --exit-code -- "$@"
+
+    local regex=$(join_to_regex "$@")
+    regex=${regex::-1}
+    local untracked_files=$(git ls-files --others --exclude-standard)
```

Comment:
`git ls-files` takes files as varags. Can you not just pass the list from the args to ls-files instead of joining them in a regex and using grep?

Reply:
that actually works for me. I think you can simplify the script to something like 
`git ls-files --others --exclude-standard "$@"`

Reply:
hmm, need to test  if it properly descends directories